Content.IntegrationTests/Tests/Round/CharacterSelectionTest.cs

```diff
@@ -256,14 +242,17 @@ public Dictionary<ProtoId<JobPrototype>, JobPriority> MakeJobPrioDict()
         // BUT the behaviour described there was later changed as described in case 1 in
         // https://github.com/space-wizards/space-station-14/pull/36493#issuecomment-3014257219,
         // so this tests the updated behaviour
+        // Since this player has no enabled characters that are eligible to be a traitor, their session will not be
+        // selector to be an antag, and thus the Captain character will be eligible to spawn, even if the Captain
```

Comment:
Suggestions:
selector -> selected
"even if the Captain character has traitor enabled" -> "and will not be traitor due to having no traitor-compatible jobs."